src/utils/useSubset.jsx

```diff
@@ -70,6 +70,12 @@ export const useSubset = (init =  {
             case 'classifications_remove_excluded': {
                 return  {...state, classifications: state.classifications.filter(c => c.included)};
             }
+            case 'classification_remove_all_codes': {
+                state.classifications.forEach(code => code.codes?.forEach(codes => {
+                    if(codes.hasOwnProperty('included')) {delete codes.included;}
```

Comment:
Nice. It keeps the chosen classification in the state and codes overwrite with codes for the new dates. 
Very good 👍